python/src/aac/__init__.py

```diff
@@ -13,7 +13,7 @@
 import os
 
 
-__version__ = "0.3.15"
+__version__ = "0.3.16"
```

Comment:
Per our discussion, we should revert the version bump.

```suggestion
__version__ = "0.3.15"
```